python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -167,6 +167,12 @@ class PlotImageSubtractionCutoutsConfig(pexConfig.Config):
         default=10000,
         optional=True
     )
+    save_raw_img = pexConfig.Field(
+        doc="Save the raw cutout images in numpy format.",
+        dtype=bool,
+        default=False,
+        optional=True
```

Comment:
I don't think you need to declare this as optional--just leave it default `False` and then you don't have to check if it is defined.

Reply:
Removed the `optional` part